backend/drm/drm.c

```diff
@@ -672,25 +673,39 @@ bool drm_connector_set_mode(struct wlr_output *output,
 	return true;
 }
 
-bool wlr_drm_connector_add_mode(struct wlr_output *output,
+static bool drm_connector_set_custom_mode(struct wlr_output *output,
+		int32_t width, int32_t height, int32_t refresh) {
+	drmModeModeInfo mode = {0};
+	generate_cvt_mode(&mode, width, height, (float)refresh / 1000, false, false);
+	mode.flags |= DRM_MODE_TYPE_USERDEF;
```

Comment:
_I know this is listed as WIP, but I tried it anyway_
This needs a `s/flags/type/` to avoid the early return in `wlr_drm_connector_add_mode`

Reply:
Indeed! Fixed